deploy/osd-26960/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
+    - key: hive.openshift.io/version-major-minor
+      operator: In
+      values: ["4.11", "4.12", "4.13", "4.14", "4.15", "4.16", "4.17"]
```

Comment:
I just copied what was done in a previous example of targeting non-HCP. Unsure if this the preferred approach or not.